src/test/scala/org/hammerlab/guacamole/main/GeneratePartialFasta.scala

```diff
@@ -77,15 +80,6 @@ object GeneratePartialFasta extends Logging {
       })
     })
     writer.close()
-    Common.progress("Wrote: %s".format(args.output))
-  }
-
-  private def printUsage() = {
```

Comment:
Sent out #451 to address running these while leaving them under `test`, so I think this is resolved.